packages/paste-website/src/components/AspectRatio.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {Absolute} from '@twilio-paste/absolute';
+import {Box} from '@twilio-paste/box';
+
+interface AspectRatioProps {
+  aspectRatio: string;
+  children: NonNullable<React.ReactNode>;
+}
+
+const RATIO_REGEX = /^(\d+:\d*)$/;
+
+const isCorrectPattern = (ratio: string): boolean => RATIO_REGEX.test(ratio);
+
+const handlePropValidation = ({aspectRatio}: AspectRatioProps): void => {
+  const hasRatio = aspectRatio != null && aspectRatio !== '';
+
+  if (!hasRatio) {
+    throw new Error(`[Paste: AspectRatio] Missing 'aspectRatio' prop.`);
+  }
+
+  if (!isCorrectPattern(aspectRatio)) {
+    throw new Error(`[Paste: AspectRatio] 'aspectRatio' is invalid. Use a colon separated number pattern (4:3).`);
+  }
+};
+
+const AspectRatioContainer = styled(Box)`
```

Comment:
Do you even need to extend box here? What are you using from it?

Reply:
I though whenever we needed a `div`, we should be using our `Box` component. I'm not extending anything from it, so I can switch it back to a `div`.

Reply:
I think when you actually want to use some of the style props, or expose them to a consumer, extend away. If you're not using it, then a div is fine. Save dependencies and when it comes to paste packages, save bloating the package too if it only needs a div

Reply:
👍